examples/client/beatBuilder.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<head>
+    <title>Crunker test</title>
+    <script src="https://unpkg.com/crunker@latest/dist/crunker.js"></script>
+    <script type="text/javascript">
+        mergeOneLine = async function(crunker, buffer, times){
+            let audios = times.map(t => crunker.padAudio(buffer, 0, t))
+            return crunker.mergeAudio(audios)
+        }
+        
+        beatBuilder = async function(crunker, beatConf, buffers, beatTiming){
+            let beatLines = await Promise.all(buffers.map((_,i) => {
+                return mergeOneLine(crunker, buffers[i], beatTiming[i].map(bt => bt*beatConf.timing))
+            }))
+            let beat = await crunker.mergeAudio(beatLines)
+            return await mergeOneLine(crunker, beat, Array.from(Array(beatConf.repeats).keys()).map(e => beatConf.delayStart+e*beatConf.beats*beatConf.timing))
+        }
+        
+        window.onload = async function(){
+            let crunker = new Crunker.default({sampleRate:96000});
```

Comment:
`let` can be const, always prefer const over let as it more explicitly shows that the variable cannot be overriden and helps make this code easier to read because a const declaration value can never be changed, whereas with let I have to keep reading expecting it to be overriden.